docs/analysis/sen-be-007-verify-e2e.md

```
@@ -0,0 +1,74 @@
+
+```bash
+for n in 0 1 2 3 4; do
+  status=$(curl -sI "https://smartlic.tech/sitemap-${n}.xml" | head -1 | tr -d '\r')
+  count=$(curl -s "https://smartlic.tech/sitemap-${n}.xml" | grep -c "<url>")
+  echo "sitemap-${n}.xml: ${status} | <url>=${count}"
+done
+```
+
+Expected:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)